DeusEx/Classes/DXREntranceRando.uc

```diff
@@ -127,6 +127,7 @@ function bool IsDeadEnd(string mapname)
 
 function bool CanSelfConnect(string mapname)
 {
+    return false;
     return GetNumXfersByMap(mapname)>2;
```

Comment:
I don't see a point to making a self-connecting map, and making a map self-connecting means that you're taking away teleporters that are normally use to go to a different map, which means now you have teleporters that aren't used

Reply:
Allowing self-connections opens up more possibilities for deeper map chains, eg NYC Streets>Bar>Sewers>Hotel>NSF, where most of the remaining streets teleporters would link to themselves.  I think it's interesting, but maybe it could just be an option?

Reply:
yea I see what you mean, I'm gonna focus on getting stuff working and avoiding softlocks before enabling self-connections though